lib/discourse.php

```diff
@@ -200,10 +200,10 @@ function sso_parse_request( $wp )
   function get_avatar_url( $user_id ) {
     $avatar = get_avatar( $user_id );
     $doc = new DOMDocument();
-    $doc->loadHTML( $avatar );
+    $doc->loadHTML( mb_convert_encoding( $avatar, 'HTML-ENTITIES', 'UTF-8' ) );
     $xpath = new DOMXPath( $doc );
     $src = $xpath->evaluate( 'string(//img/@src)' );
-    return $src;
+    return utf8_uri_encode($src);
```

Comment:
can you please update this line to be consistent with the other formatting? (space after opening paren, space before closing paren)